code/scripting/api/libs/mission.cpp

```diff
@@ -1387,13 +1387,7 @@ ADE_FUNC(isInCampaignLoop, l_Mission, nullptr, "Get whether or not the current m
 
 ADE_FUNC(isTraining, l_Mission, nullptr, "Get whether or not the current mission being played is a training mission", "boolean", "true if in training, false if not")
 {
-	bool b = false;
-
-	if (The_mission.game_type & MISSION_TYPE_TRAINING) {
-		b = true;
-	}
-
-	return ade_set_args(L, "b", b);
+	return ade_set_args(L, "b", !!(The_mission.game_type & MISSION_TYPE_TRAINING));
```

Comment:
Hmm, would it not be easier to read if we just properly cast to a bool using either a c-style cast or a static cast?

Reply:
I suppose so.